src/coreclr/vm/typedesc.h

```diff
@@ -163,6 +163,25 @@ class TypeDesc
 
     BOOL ContainsGenericVariables(BOOL methodOnly);
 
+#ifndef DACCESS_COMPILE
+    OBJECTREF GetManagedClassObject();
+
+    OBJECTREF GetManagedClassObjectIfExists() const
+    {
+        CONTRACTL
+        {
+            NOTHROW;
+            GC_NOTRIGGER;
+            MODE_COOPERATIVE;
+        }
+        CONTRACTL_END;
+
+        const RUNTIMETYPEHANDLE handle = m_hExposedClassObject;
+        OBJECTREF retVal = ObjectToOBJECTREF((Object*)handle);
```

Comment:
```suggestion
        OBJECTREF retVal = ObjectToOBJECTREF(handle);
```
RUNTIMETYPEHANDLE can be defined as `Object*` to save casts like this. It is actually Object* now.